src/DssSpell.t.sol

```diff
@@ -440,48 +479,6 @@ contract DssSpellTest is DssSpellTestBase {
             _claimed:    0                                               // rxd
         });
 
-        assertTrue(vest.valid(11)); // check for valid contract
```

Comment:
nit: from last week's review I think @godsflaw wanted to keep those for future use

Reply:
This exists on line 468.

Reply:
yeah, it has been trimmed though, I think consensus was to keep the rest as well, but as you pref.

Reply:
as for the previous comment, this could be generalised and moved to base tests as well.